src/main/java/com/ezasm/gui/RegisterTable.java

```diff
@@ -44,6 +48,27 @@ public RegisterTable(Registers registers) {
         add(scrollPane);
     }
 
+    /**
+     * The function to highlight changed registers      
+     */
+    public void ChangeCellColor(Config config) {
+        Theme theme = Theme.getTheme(config.getTheme());
```

Comment:
getting the theme every time seems weird. Why not store the colors we need when we call `applyTheme`?